scene/main/scene_tree.cpp

```diff
@@ -407,8 +407,8 @@ void SceneTree::set_group(const StringName &p_group, const String &p_name, const
 void SceneTree::initialize() {
 	ERR_FAIL_COND(!root);
 	initialized = true;
```

Comment:
Seems like this `initialized` member isn't used for anything (it's being assigned `true`/`false` in `initialize`/`finalize` and that's all). Probably it could be just removed? :thinking:

If not removed then it's strange that `initialized = true` is set at the beginning of the initialization instead of at the end.

Reply:
Yeah, looks like `initialized` is legacy param and can be safely deleted.